src/meta_schedule/postproc/postproc.cc

```diff
@@ -50,6 +50,48 @@ Postproc Postproc::PyPostproc(
   return Postproc(n);
 }
 
+Array<Postproc> Postproc::DefaultLLVM() {
+  return Array<Postproc>{
+      Postproc::DisallowDynamicLoop(),
+      Postproc::RewriteParallelVectorizeUnroll(),
+      Postproc::RewriteReductionBlock(),
+      Postproc::RewriteLayout(),
+  };
+}
+
+Array<Postproc> Postproc::DefaultCUDA() {
+  return Array<Postproc>{
+      Postproc::DisallowDynamicLoop(),
+      Postproc::RewriteCooperativeFetch(),
+      Postproc::RewriteUnboundBlock(/*max_threadblocks=*/256),
+      Postproc::RewriteParallelVectorizeUnroll(),
+      Postproc::RewriteReductionBlock(),
+      Postproc::VerifyGPUCode(),
+  };
+}
+
+Array<Postproc> Postproc::DefaultCUDATensorCore() {
+  return Array<Postproc>{
+      Postproc::DisallowDynamicLoop(),
+      Postproc::RewriteCooperativeFetch(),
+      Postproc::RewriteUnboundBlock(/*max_threadblocks=*/256),
+      Postproc::RewriteParallelVectorizeUnroll(),
+      Postproc::RewriteReductionBlock(),
+      Postproc::RewriteTensorize(/*vectorize_init_loop=*/false),
+      Postproc::VerifyGPUCode(),
+  };
+}
+
+Array<Postproc> Postproc::DefaultHexagon() {
+  return Array<Postproc>{
+      Postproc::DisallowDynamicLoop(),
+      Postproc::RewriteParallelVectorizeUnroll(),  //
+      Postproc::RewriteReductionBlock(),
+      // TODO(masahi): Fix RewriteLayout for link-params=True case
+      // Postproc::RewriteLayout(),
+  };
+}
```

Comment:
I moved the default_config._DefaultHexagon.postprocs to C++, would you like to take a second look in case anything goes wrong?

Reply:
Packed func registration and the corresponding Python definition are missing for Hexagon. Same for schedule_rule / mutator.

Reply:
got it. done!